src/renderer/viz/expressions/basic/property.js

```diff
@@ -71,20 +71,23 @@ export default class Property extends BaseExpression {
     }
 
     _bindMetadata (metadata) {
-        const metaColumn = metadata.properties[this.name];
-        if (!metaColumn) {
-            throw new Error(`Property '${this.name}' does not exist`);
-        }
+        if (metadata.properties) {
```

Comment:
Maybe we could just end early, to avoid a big if
```js
if (!metadata.properties){
  return;
}
```

Reply:
Actually, I don't see why is this `if` in the first place. If there are no properties defined in the metadata this should fail hard, not silently ignore the problem (I think)